Sources/SpeziConsent/Model/ConsentDocument.swift

```diff
@@ -0,0 +1,411 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import PencilKit
+import SpeziFoundation
+import SwiftUI
+import class PDFKit.PDFDocument
+
+
+/// Represents and manages a Markdown-based (potentially interactive) Consent Document.
```

Comment:
Thank you for adding all these docs!

Did we update all existing DocC articles in the repo?